src/workflow/OSWorkflow.hpp

```diff
@@ -6,6 +6,9 @@
 #include "../utilities/core/Filesystem.hpp"
 #include "../utilities/filetypes/WorkflowJSON.hpp"
 
+#define USE_RUBY_ENGINE 1
```

Comment:
Are these configured? I didn't notice any cmake reference to them. Just wondering. I'm going to merge this regardless.